actionpack/lib/action_controller/metal/allow_browser.rb

```diff
@@ -13,8 +13,9 @@ module ClassMethods
       # versions specified. This means that all other browsers, as well as agents that
       # aren't reporting a user-agent header, will be allowed access.
       #
-      # A browser that's blocked will by default be served the file in public/426.html
-      # with a HTTP status code of "426 Upgrade Required".
+      # A browser that's blocked will by default be served the file in
+      # public/406-unsupported-browser.html with a HTTP status code of "406 Not
```

Comment:
I chose not to use `public/406.html` because 406 is often used for more generic cases (e.g. JSON content negotiation) and we don't want to confuse the two use cases.

Reply:
426 is wrong, so it's good that this was changed to something else. 406 with the information supplied is fine. I do a lot of content negotiation, and it's almost never automatically resolvable. Did you have cases that are different?

Reply:
I think that's smart. I do a lot of content negotiation, and it's almost never automatically resolvable -- and we have 406.html implemented for use cases you mention ;)